rbac/rbac/middleware.py

```diff
@@ -85,23 +82,7 @@ def get_tenant(self, model, hostname, request):
                 except Tenant.DoesNotExist:
                     raise Http404()
             else:
-                tenant, created = Tenant.objects.get_or_create(schema_name=tenant_schema)
-                if created:
-                    try:
-                        with transaction.atomic():
-                            tenant.create_schema(check_if_exists=True)
-                            seed_permissions(tenant=tenant)
-                            seed_roles(tenant=tenant)
-                            seed_group(tenant=tenant)
-                            tenant.ready = True
-                            tenant.save()
-                    except Exception as e:
-                        tenant.delete()
-                        raise e
-                else:
-                    while not tenant.ready:
-                        time.sleep(0.5)
-                        tenant.refresh_from_db()
+                tenant, created = Tenant.objects.get_or_create(schema_name=tenant_schema, defaults={"ready": True})
```

Comment:
Good call on having the default here, vs a keyword arg which would try to create in the case of "stuck" tenants. 👍